### PR TITLE
feat(apis): improve infra code

### PR DESCRIPTION
**Pull request by shinytang6**
Signed-off-by: shinytang6 <1074461480@qq.com>

### CODE REVIEW

**Comment by Thor-wl** [comment]
@shinytang6 Can you help about the CI settings?

File: hack/update-codegen.sh [hunk 1]
```diff
@@ -6,8 +6,7 @@ set -o pipefail
 
 SCRIPT_ROOT=$(dirname ${BASH_SOURCE})/..
 CODEGEN_PKG=${CODEGEN_PKG:-$(cd ${SCRIPT_ROOT}; ls -d -1 ./vendor/k8s.io/code-generator 2>/dev/null || echo ${GOPATH}/src/k8s.io/code-generator)}
-
-vendor/k8s.io/code-generator/generate-groups.sh "deepcopy,client,informer,lister" \
+bash $CODEGEN_PKG/generate-groups.sh "deepcopy,client,informer,lister" \
   volcano.sh/apis/pkg/client volcano.sh/apis/pkg/apis \
```
Comment (k82cn) [inline hack/update-codegen.sh:10]: which branch of apis do we use?

Comment (shinytang6) [reply hack/update-codegen.sh:10]: l used master branch in this PR

Comment (k82cn) [reply hack/update-codegen.sh:10]: it's better to make it configurable; if someone, checkout this script in different branch, it'll also use master to generate client-go :)

Comment (shinytang6) [reply hack/update-codegen.sh:10]: l don't really catch your mean. If user checkout to other branch and use this script, the client will be generated according to the apis in that branch, while not master

**Comment by Thor-wl** [comment]
I've updated the changes you make in the latest PRs. Thanks for the contribution.

**Pull request closed without merging**